targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_062_BLE/device/system_psoc6_cm4.c

```diff
@@ -43,6 +43,10 @@
     #endif /* defined(CY_DEVICE_PSOC6ABLE2) */
 #endif /* !defined(CY_IPC_DEFAULT_CFG_DISABLE) */
 
+#if defined(COMPONENT_SPM_MAILBOX)
```

Comment:
please remove this code until you actually add the BLE kit as a PSA target

Reply:
See https://github.com/ARMmbed/mbed-os/pull/10308#discussion_r275443336

Reply:
Updated readme.me in the https://github.com/ARMmbed/mbed-os/pull/10308/commits/83275da0cf4d25a4b8da3d9ed7f195188e76de69 commit